templates/terraform/examples/artifact_registry_repository_cmek.tf.erb

```diff
@@ -0,0 +1,9 @@
+resource "google_artifact_registry_repository" "<%= ctx[:primary_resource_id] %>" {
+  provider = google-beta
+
+  location = "us-central1"
+  repository_id = "<%= ctx[:vars]['repository_id'] %>"
+  description = "<%= ctx[:vars]['description'] %>"
```

Comment:
vars are typically used for things that need to have different values in the test compared to the example, but I think the description can be the same in both places. You can see in the generated code at https://github.com/modular-magician/terraform-provider-google-beta/compare/auto-pr-3658-old..auto-pr-3658#diff-9fbc35188620ea4ba754beea15012972R84 what happens when you have it as a var- it puts a random suffix at the end (which is important for things like names in tests because two people might run the same test in parallel and we don't want a conflict)